### PR TITLE
Updates to `FindVoro++.cmake`

### PR DESCRIPTION
**Pull request by js947**
cmake now finds voro++ without help!

### CODE REVIEW

**Comment by js947** [comment]
Also tested on darwin with voro++ installed as a environment module.

**Comment by js947** [comment]
At this time, travis has green flags on all of the linux builds but the mac builds aren't starting...

File: CMakeLists.txt [hunk 1]
```diff
@@ -60,11 +60,8 @@ SET(lem_src
 add_executable(lem
                ${lem_src}
 )
+target_link_libraries(lem ${VOROPLUSPLUS_LIBRARIES})
 
-target_link_libraries(lem voro++)
-
-option(VOROPLUSPLUS_HEADER_PATH  "path to voro++ header files,  command line option"  "")
```
Comment (kks32) [inline CMakeLists.txt]: Won't I still need this, if I have to pass the install path for Voro++?

Comment (kks32) [inline CMakeLists.txt]: This removes the possibility of passing location as an option to CMake?

**Comment by kks32** [comment]
Doesn't this remove the possibility of passing path to `voro++` as an option to CMake?

**Comment by js947** [comment]
if the find_path or find_library functions fail you can give cmake the paths by
    cmake ~/lem -DVOROPLUSPLUS_INCLUDE_DIR=/blah/voro++/include -DVOROPLUSPLUS_LIBRARIES=/blah/voro++/lib/libvoro++.a

**Comment by kks32** [comment]
That's great! Thanks.

**Pull request merged**